packages/terra-list/src/SingleSelectList.jsx

```diff
@@ -123,22 +123,13 @@ class SingleSelectList extends React.Component {
       newProps.isSelected = isSelected;
     }
 
-    // By default isSelectable attribute for the Item is undefined, as this is selectable list,
-    // we will make item selectable by default. If consumer specify the row attribute isSelectable as false,
-    // then the item will not be selectable
-    const isSelectable = item.props.isSelectable;
-    if (isSelectable === undefined) {
-      newProps.isSelectable = true;
-    }
-
+    newProps.isSelectable = item.props.isSelectable;
     // Add tabIndex on items to navigate through keyboard tab key for selectable litst
-    if (newProps.isSelectable || isSelectable) {
+    if (newProps.isSelectable) {
       newProps.tabIndex = '0';
     }
 
-    if (item.props.hasChevron === undefined) {
-      newProps.hasChevron = this.props.hasChevrons;
-    }
+    newProps.hasChevron = this.props.hasChevrons;
 
     return newProps;
```

Comment:
I believe the change to delete the "onChange" and "maxSelectionCount" in mutli select, and 'onChange' and 'hasChevrons' in single select should not have been made. That is directly manipulation the props and we are losing the onChange event passed into the component by the user. A better solution would be to use:
// eslint-disable-line no-unused-vars
when we are unpacking the customProps.